modules/userId/index.js

```diff
@@ -104,6 +104,7 @@ import CONSTANTS from '../../src/constants.json';
 import {module} from '../../src/hook';
 import {unifiedIdSubmodule} from './unifiedIdSystem.js';
 import {pubCommonIdSubmodule} from './pubCommonIdSystem.js';
+import {netIdSubmodule} from './netIdSystem.js';
```

Comment:
Please remove this line -- unified and pubcommon are the only two in this list for grandfathered reasons. They will be removed in Prebid.js 3.0. Every user-id sub-module needs to be separately specified in the `gulp build` command.